### PR TITLE
FAI-1850 Account Deletion Validation Page

### PR DESCRIPTION
**Pull request by balaji-j**
https://skillsfundingagency.atlassian.net/browse/FAI-1850

### CODE REVIEW

File: src/SFA.DAS.FAA.Web.UnitTests/Controllers/Users/WhenGettingAccountDeletion.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        {
+            var result = controller.AccountDeletion(journeyPath) as ViewResult;
+            result.Should().NotBeNull();
+
+            var actualModel = result!.Model as AccountDeletionViewModel;
+
+            actualModel.Should().NotBeNull();
+            actualModel!.JourneyPath.Should().Be(journeyPath);
+            actualModel.PageBackLink.Should().Be(pageBackLink);
+            actualModel.IsAnyOutstandingApplications.Should().Be(isAnyOutstandingApplications);
```
Comment (dashton82) [inline src/SFA.DAS.FAA.Web.UnitTests/Controllers/Users/WhenGettingAccountDeletion.cs:35]:
```suggestion
            actualModel.HasAnyOutstandingApplications.Should().Be(hasAnyOutstandingApplications);
```
I think the model must be in another PR but could that property be renamed to the above please?

Comment (balaji-j) [reply src/SFA.DAS.FAA.Web.UnitTests/Controllers/Users/WhenGettingAccountDeletion.cs]: Renamed Property name

**Review by dashton82 — APPROVED** [review]

**Review by dashton82 — APPROVED** [review]
